blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/Discovery.java

```diff
@@ -24,6 +24,9 @@
 import java.util.Map;
 
 public class Discovery {
+
+    protected final static String SEPARATOR = ":";
```

Comment:
Yes, but `Discovery` and `Token` are these separate implementations. I thought I will keep them separate.